src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
```

Comment:
As I remember header doesn't have default margins. Why you do you need to reset them? Please check